platform/engineering/design-docs/2020-05-26-content-validation.md

```diff
@@ -139,19 +172,50 @@ If invalid content is breaking the production build, and therefore blocking the
+
+For content writers, this approach preserves their ability to quickly draft, publish, and deploy content. And the improved reporting will make it easier for them to pinpoint errors. The downside of this approach is that content writers still won't have a proactive way to fully validate their content.
 
-The Metalsmith static content build will be extracted to the `content-build` repo and deleted from `vets-website`, and its deployment pipeline. Webpack will handle the application build for the `vets-website` repo, and Metalsmith will handle the static content build for the `content-build` repo.
+For the Frontend Tools team, this approach will reduce noise in the application build logs, and eliminate a source of failed application builds. Eliminating a source of failed application builds means the Frontend Tools team will have to spend less time manually triaging failed application builds, which will give the team more bandwidth to work on other priorities.
 
-The validation of static content will take place in a scheduled job that runs every workday instead of ???. Since the broken link check happens during the Metalsmith build, and the accessibility check relies on the output of the content build, the scheduled job will run a script to produce a build that only has the steps necessary for content validation. That build will only be used for reporting broken links and accessibility errors, and will have no impact on deploys. Those reports will be delivered to the #cms-team channel via existing Slack integrations.
+For anybody trying to ship code for the VA.gov front end, this approach will eliminate a source of failed application builds, which should allow them to ship code faster.
+
+For VA.gov users, this approach will continue allowing users to quickly see updated content. The downside of this approach for VA.gov users is that it won't eliminate existing opportunities for users to see invalid content. Fortunately, the improved error reporting should allow content writers to easily identify, remedy, and deploy valid content, which should minimize the user's chance of encountering invalid content.
+
+In summary, this approach will have the following impact for the following groups:
+
+- **Content Writers**: Improve error reporting while maintaining current convenience
+- **Frontend Tools team**: Reduce build noise, failure rate, and number of builds requiring manual intervention
+- **VA.gov users**: Maintain existing time to see updated content and risk of seeing invalid content
```

Comment:
hmm, is this true? my understanding was that broken link checking currently gates deployment, so if we move it to a daily job that _doesn't_ gate deployment, doesn't that increase risk of seeing invalid content?

Reply:
Your understanding is correct. I apologize for the confusion. The current system does block deploys when broken links are detected in `master`. I've updated the design doc to clarify this in https://github.com/department-of-veterans-affairs/va.gov-team/pull/9658/commits/c3389e0b45f31d14079cc1a8d26a898d0a280de8.